index.html

```diff
@@ -1,14 +1,82 @@
 <!DOCTYPE html>
-<html>
-<head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Page Title</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" type="text/css" media="screen" href="styles.css">
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>HTML Forms</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" media="screen" href="styles.css" />
     <script src="main.js"></script>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <!-- NAV -->
+    <nav>
+      <div class="navbar-container">
+        <div class="navbar-text">
+          <div>Already have an Intuit account?</div>
+          <a href="#">I forgot my User ID or Password</a>
+        </div>
+        <div class="navbar-button"><button>Sign In</button></div>
+      </div>
+    </nav>
+
+    <!-- HEADER -->
+    <header>
+      <h1>INTUIT</h1>
+      <div id="headerBrands">
+        <img src="./assets/mint.svg" alt="mint" /> <a href="#">mint</a>
+        <img src="./assets/qb.svg" alt="quickbooks" />
+        <a href="#">quickbook</a> <img src="./assets/tt.svg" alt="turbotax" />
+        <a href="#">turbotax</a>
+      </div>
+    </header>
+
+    <!-- FORM BODY -->
+    <div class="form-container">
+      <h2>One Account.<br />Everything Intuit.</h2>
+      <h3>
+        Create an account for all of Intuit's products including Mint.
+        <a href="#">Learn more</a>
+      </h3>
+      <form>
+        <label>
+          Email address <input type="email" name="email" required />
+        </label>
+        <label> Phone (recommended) <input type="email" name="email" /> </label>
+        <span>Standard call, messaging or data rates may apply.</span>
+        <label>
+          Password <input type="password" name="password" required />
+        </label>
```

Comment:
Kindly check use of `label` and `input` [here](https://www.w3schools.com/tags/tag_label.asp) and refactor code accordingly. This may not be caught in HTML5 validator but it'll break in older browsers.